packages/mui-material-next/test/describeConformance.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  describeConformance as baseDescribeConformance,
+  ConformanceOptions,
+} from '@mui-internal/test-utils';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
```

Comment:
For the record, just in case another reviewer wonders:

For most components in the `material-next` package, we're overriding these with [`CssVarsProvider`](https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/styles/CssVarsProvider.tsx) and [`extendTheme`](https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/styles/extendTheme.ts) from the `material-next` package. For example: https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/ButtonBase/ButtonBase.test.tsx#L58

Using `CssVarsProvider` and `extendTheme` as default would break the tests on the `material-next` package before we started the Material Design 3 implementations. So this is ok.

After v7, we must clean (remove?) the `material-next` package.

Reply:
I think for testing purpose, `ThemeProvider` makes more sense than `CssVarsProvider` because we don't need CSS vars generation in most tests.

Note: `CssVarsProvider` uses `ThemeProvider` internally